r/R/utilities.R

```diff
@@ -2,28 +2,64 @@
 # subsetIds subsets a seurat object with the cell ids
 #
 subsetIds <- function(scdata, cells_id) {
-  meta_data_subset <- scdata@meta.data[match(cells_id, scdata@meta.data$cells_id), ]
+  meta_data_subset <-
+    scdata@meta.data[match(cells_id, scdata@meta.data$cells_id), ]
   current_cells <- rownames(meta_data_subset)
   scdata <- subset(scdata, cells = current_cells)
   return(scdata)
 }
 
-sendCellsetToApi <- function(new_cell_set, api_url, experiment_id, cell_set_key, auth_JWT) {
-  httr_query <- paste0('$[?(@.key == "', cell_set_key, '")]')
-  children <- list(list("$insert" = list(index = "-", value = new_cell_set)))
+sendCellsetToApi <-
```

Comment:
little nit-pick, but can you have a description about what this method does and its parameters for `sendCellsetToApi` similarly to how we have for the other functions in this file?

Reply:
Also, can you fix the naming inconsistency? In this function name, we have `Cellset`. In `updateCellSetsThroughApi` we have `CellSets`. Can we use the same convention?

Reply:
Added documentation to those functions. Did not change the cell set to cell setS because that function works with a single CellSet.